airbyte-integrations/connectors/source-bing-ads/source_bing_ads/reports.py

```diff
@@ -111,6 +111,7 @@
     "ExactMatchImpressionSharePercent": "number",
     "Impressions": "integer",
     "ImpressionSharePercent": "number",
+    "Keyword": "string",
```

Comment:
Adding a string type here has no sense because it is ignored in https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-bing-ads/source_bing_ads/reports.py#L296